app/views/categories/show.html.haml

```diff
@@ -0,0 +1,58 @@
+.l-panel-content.l-panel-content--form
+  .l-constrained
+    .l-panel-content__row
+      .l-panel-content__col
+        %h1 Category Manager
+      .l-panel-content__col
+
+.l-panel-content
+  .l-constrained
+    %h2.breadcrumb
+      - @category.parents.each do |category|
+        = link_to category.label, category_path(category)
+        >
+      =@category.label
+
+    .l-panel-content__row
+      .l-panel-content__row
+      .l-split-content
+        .l-split-content__col
+          %h2 Category details
+          = comfy_form_for @category, url: {action: "update"}, html: {class: 'form form--vertical'}, label_col: "form__label-heading", control_col: "form__input-wrapper" do |f|
+            = f.text_field :title_en
+            = f.text_field :title_cy
+            = f.text_field :description_en
+            = f.text_field :description_cy
+
+            .form-group
+              .form__input-wrapper
+                = f.check_box :navigation
+
+            = hidden_field_tag 'list_order_sub_categories', nil , data: { dough_listsorter_order_field: true, dough_listsorter_context: 'catman-sub-categories'}
+            = hidden_field_tag 'list_order_pages', nil , data: { dough_listsorter_order_field: true, dough_listsorter_context: 'catman-pages'}
```

Comment:
Can we replace *catman* with category-manager or category or just manager (is up to you).?